server/action/video/update.go

```diff
@@ -286,6 +278,70 @@ func update(w http.ResponseWriter, r *http.Request) {
 		meiliPublishDate = videoObj.PublishedDate.Unix()
 	}
 
+	var updatedOrCreatedVideoBlock []uint
+
+	ratingIds := make([]uint, 0)
+	claimantIds := make([]uint, 0)
+	rMap := make(map[uint]uint)
+	cMap := make(map[uint]uint)
+
+	for _, each := range videoData.Claims {
```

Comment:
What is use of rMap and cMap. Can't we add directly to ratingIds & claimantIds in one loop.